### DISCUSSION TITLE
fhirStore persistance?

### DISCUSSION THREAD

**Discussion by advatar** [StanfordSpezi/.github#67] [Challenges and Help] [answered]
### What Stanford Spezi module is your challenge related to?

SpeziFHIR

### Description

What is the idea for persistance? Using FHIR Bundle on disk?

### Reproduction

noticed that fhirStore starts empty on restart

### Expected behavior

would like fhirStore to persist

### Code of Conduct

- [X] I agree to follow this projects's Code of Conduct and Contributing Guidelines

**Comment by PSchmiedmayer** [comment, marked as answer]
Thank you for reaching out @advatar!

Yes, persisting a bundle on disk might be the simplest way.

If you want to have a more scalable local storage backend, I would recommend Swift Data storing some context in a field and the complete resource in a JSON blob.
